### ISSUE TITLE
Add locale translations

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/nta-arclight#11]
Add locale translations into UI
To understand how big a task this is, we need to know how the current templates in AL work, and if there are inline strings we'd need to swap out with locale YAML.

**Comment by marlo-longley** [comment]
See

https://github.com/projectblacklight/arclight/issues/594
https://github.com/projectblacklight/arclight/issues/611

**Comment by jcoyne** [comment]
@marlo-longley which locales are we targeting?  Do we have all of the metadata available in all locales?

**Comment by marlo-longley** [comment]
@jcoyne No, we definitely only have English metadata. As I learn more about the product I think this ticket is actually 2 things:

1. Go through the Arclight templates and make sure they use locale rather than hardcoded text. So that ticket should live in Arclight repo. It is debatable how high a priority this is for this workcyclce. But it's good cleanup work in general.

2. Customize text in NTA-Arclight as needed with translations. The need for this is TBD from @laurensorensen. We have transcripts of A/V in multiple languages, but how those will be accessed/displayed, and if there will need to be translations within the app to support them, is TBD.

**Comment by ggeisler** [comment]
@marlo-longley Might we be missing a step between steps 1 and 2 in your comment above? Step 1 sets Arclight up to effectively provide Arclight UI strings in non-English locales, but we also need to create and populate the non-English locale files so those string translations actually exist in the non-English locales (which will require a non-development step of finding translators for each of the target locales).

I suggest that is a step between your 2 steps because I don't think we should worry too much about your second ssetep of providing NTA-specific translations unless we're sure we will be offering non-English options in core Arclight (i.e., I'm not sure it would be a good user experience to offer German translations of NTA-specific things if the basic core Arclight UI remains in English).

(The two Arclight tickets you reference above do discuss issues around adding new locales to Arclight but they're both closed. I'm thinking we should have an open ticket related to this, if we decide translations are part of our goals for the current workcycle -- FWIW, Hannah indicated this is a high-priority feature, although probably not as high as the more fundamental UI updates, etc.)

**Comment by ggeisler** [comment]
@marlo-longley I might have time to work on the core Arclight locale stuff during the workcycle so we're prepared to support non-English locale files. And @laurensorensen and I can work on determining whether we could get appropriate translations or not. So we could works towards supporting non-English locales in Arclight, but not commit to doing it if we can't find translators, and not commit any developer time to it.

The NTA-specific translations I think could be relevant, aside from the AV transcripts, are any static text specific to NTA that we display in the NTA application. So things like homepage text, about page text, etc. (e.g., if a user can switch the locale of the NTA app to German and see German language UI elements, it would be nice if they also saw German text on the homepage instead of English). But that is independent and secondary to supporting other languages in core Arclight, I believe.

**Comment by laurensorensen** [comment]
Blocked due to lack of time in this workcycle for facilitating the 100s of fields involved in locale translation. Leaving for a future workcycle.
